spec/system/index_spec.rb

```diff
@@ -33,10 +33,6 @@
                                     text: 'Stop -1 not found'
     end
   end
-  context 'without completing stop id' do
```

Comment:
Don't know why I thought this was a functionality of the index page. Just to be sure, this isn't something that we were hoping to implement, correct?